src/classes/COFF_Affiliation_TDTM.cls

```diff
@@ -471,20 +489,20 @@ public class COFF_Affiliation_TDTM extends TDTM_Runnable {
     }
 
    /*******************************************************************************************************
-    * @description Determines whether an Affiliation to the Department the Course Offering belongs to already
-    * exists for each faculty member. Public only so it can be called from the tests.
+    * @description Determines whether an Affiliation to the Account the Course Offering belongs to already
```

Comment:
I won't hold the pull request for this, but just fyi, the term "Department" was being used on purpose for clarity, as the lookup from Course to Account is called "Department".